adam-cli/pom.xml

```diff
@@ -14,22 +14,35 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>appassembler-maven-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
         <configuration>
-          <programs>
-            <program>
-              <mainClass>org.bdgenomics.adam.cli.ADAMMain</mainClass>
-              <id>adam</id>
-            </program>
-          </programs>
+          <createDependencyReducedPom>true</createDependencyReducedPom>
+          <filters>
+            <filter>
+              <artifact>*:*</artifact>
+              <excludes>
+                <exclude>META-INF/*.SF</exclude>
+                <exclude>META-INF/*.DSA</exclude>
+                <exclude>META-INF/*.RSA</exclude>
+              </excludes>
+            </filter>
+          </filters>
```

Comment:
OOC, I went back to the ol' days before we moved to appassembler and saw a few more exclusions (see at https://github.com/bigdatagenomics/adam/blob/adam-parent-0.6.0/adam-cli/pom.xml#L43). Might be worth doing a diff of the old `maven-shade-plugin` section versus the new one.